definitions/procedures/hammer_setup.rb

```diff
@@ -2,6 +2,9 @@ class Procedures::HammerSetup < ForemanMaintain::Procedure
   metadata do
     description 'Setup hammer'
     for_feature :hammer
+    preparation_steps do
+      Procedures::Service::Status.new(:brief => true, :failing => true)
```

Comment:
Shouldn't this be rather `Checks::ServicesUp.new`?

Reply:
Done